rero_ils/modules/acquisition/acq_accounts/api.py

```diff
@@ -106,6 +107,11 @@ def __hash__(self):
         # TODO :: find a better hash method (more complete)
         return hash(str(self.id))
 
+    @property
+    def name(self):
```

Comment:
I could agree. But as we are limited to 80 characters by line, it's useful to write `record.name` instead of `record.get('name')`, the benefit is 7 characters (mostly 10% of the max line width !)